docs/releases/1.11.1.txt

```diff
@@ -62,6 +62,8 @@ Bugfixes
 * Fixed crash when overriding the template of
   ``django.views.static.directory_index()`` (:ticket:`28122`).
 
+* Fixed unexpected data loss when saving parent object after setting on child (:ticket:`28147`).
```

Comment:
I'm a bit uncertain if this change might have unexpected consequences so I'd rather not backport it. This issue exists as far back as I checked (Django 1.8) and isn't really a critical data loss problem, I think.